pkgs/by-name/ca/cargo-aoc/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-aoc";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "gobanos";
+    repo = "cargo-aoc";
+    rev = version;
+    hash = "sha256-k9Lm91+Bk6EC8+KfEXhSs4ki385prZ6Vbs6W+18aZSI=";
+  };
+
+  cargoHash = "sha256-DKP9YMbVojK7w5pkX/gok4PG6WUjhqUdvTwSir05d0s=";
+
+  meta = {
```

```suggestion
  passthru = {
    tests.version = testers.testVersion { package = cargo-aoc; };
    updateScript = nix-update-script { };
  };
  
  meta = {
```
Nitpick, but can help make maintaining this a bit better for yourself and others (see also [versionCheckHook](https://nixos.org/manual/nixpkgs/stable/#versioncheckhook))